lib/model/internal_link.dart

```diff
@@ -96,7 +96,11 @@ Uri narrowLink(PerAccountStore store, Narrow narrow, {int? nearMessageId}) {
     fragment.write('/near/$nearMessageId');
```

Comment:
Looks like I missed this in the last round. "Variable" in the commit summary is capitalized, but we should use "variable" instead.